Makefile

```diff
@@ -1,24 +1,41 @@
 BINARY=monaco
 
-.PHONY: build clean test integration-test
+.PHONY: lint format build install clean test integration-test test-package
 
 lint:
+ifeq ($(OS),Windows_NT)
+	@.\tools\check-format.cmd
+else
 	@sh ./tools/check-format.sh
+endif
 
 format:
 	@gofmt -w .
 
-build: clean
+mocks:
```

Comment:
You need to add `mock` also to the `.PHONY` tasks

Reply:
thx, missed that one

Reply:
Fixed